msteams-platform/bots/how-to/Teams conversational AI/how-conversation-ai-get-started.md

```diff
@@ -265,6 +247,24 @@ app.ai.action(
 );
 ```
 
+### Moderation
+
+`FlaggedInputAction` and `FlaggedOutputAction` are the built-in action handlers to handle the moderator flags. If the moderator flags an incoming message input, the moderator redirects to the `FlaggedInputAction` handler and the `context.sendActivity` sends a message to the user about the flag.
```

Comment:
`FlaggedInputAction` and `FlaggedOutputAction` serve as the integrated action handlers for managing moderator flags. When a moderator flags an incoming message input, the system redirects the moderator to the `FlaggedInputAction` handler. The `context.sendActivity` then dispatches a message to the user, notifying them about the flag.

Reply:
Updated.